### PR TITLE
Publish license

### PR DESCRIPTION
**Pull request by benjaminapetersen**
based on #386 

/hold 
as the license file doesn't currently exist.

### CODE REVIEW

File: manifests/07-downloads-deployment.yaml [hunk 1]
```diff
@@ -89,10 +89,14 @@ spec:
               os.mkdir(arch)
               for operating_system in ['linux', 'mac', 'windows']:
                   os.mkdir(os.path.join(arch, operating_system))
+                  target_path = os.path.join(arch, operating_system, 'LICENSE')
+                  os.symlink('/usr/share/openshift/LICENSE', target_path)
```
Comment (jhadvig) [inline manifests/07-downloads-deployment.yaml:93]: Do we need to have a separate `LICENSE` file for each os? Can't hurt but II guess that they will be all the same.

File: pkg/console/controllers/clidownloads/controller.go [hunk 2]
```diff
@@ -188,6 +195,41 @@ The oc binary offers the same capabilities as the kubectl binary, but it is furt
...
 
+func LicenseDownloads(host, cliDownloadsName string) *v1.ConsoleCLIDownload {
+	baseURL := fmt.Sprintf("%s", util.HTTPS(host))
+	platforms := []struct {
+		label    string
+		key      string
+		archType string
+	}{
+		{"Linux for x86_64", "amd64/linux", "LICENSE"},
+		{"Linux for ARM 64", "arm64/linux", "LICENSE"},
```
Comment (jhadvig) [inline pkg/console/controllers/clidownloads/controller.go:206]:
```suggestion
		{"Linux for ARM 64 (unsupported)", "arm64/linux", "LICENSE"},
```

to be consistent with the `oc` binaries ?

File: pkg/console/controllers/clidownloads/controller.go [hunk 3]
```diff
@@ -188,6 +195,41 @@ The oc binary offers the same capabilities as the kubectl binary, but it is furt
...
+		{"Linux for IBM Power, little endian", "ppc64le/linux", "LICENSE"},
```
Comment (jhadvig) [inline pkg/console/controllers/clidownloads/controller.go:207]:
```suggestion
		{"Linux for IBM Power, little endian (unsupported)", "ppc64le/linux", "LICENSE"},
```

File: test/e2e/downloads_test.go [hunk 4]
```diff
@@ -48,3 +48,31 @@ func TestDownloadsEndpoint(t *testing.T) {
 		t.Logf("%s %s\n", link.Text, resp.Status)
 	}
 }
+
+func TestDonwloadsLicense(t *testing.T) {
```
Comment (jhadvig) [inline test/e2e/downloads_test.go:52]:
```suggestion
func TestDownloadsLicense(t *testing.T) {
```

**Comment by jhadvig** [comment]
Also do we wanna close https://github.com/openshift/console-operator/pull/386/files now ?

**Comment by benjaminapetersen** [comment]
Not sure, we have several issues that need to be addressed.

File: manifests/07-downloads-deployment.yaml (hunk 1 above)
Comment (sosiouxme) [reply manifests/07-downloads-deployment.yaml:93]: It's the same source for all, so the same license. In fact you could just use a local copy of the Apache 2.0 license and call it a day :) I don't see the license changing ever.

**Comment by jhadvig** [comment]
Closing in favour of https://github.com/openshift/console-operator/pull/439

**Comment by spadgett** [comment]
Replaced by #439

**Pull request closed without merging**